plugin/src/kiali/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -574,6 +608,7 @@ export const SummaryPanelNode: React.FC<SummaryPanelNodeProps> = (props: Summary
   const rankResult = useKialiSelector(state => state.graph.rankResult);
   const showRank = useKialiSelector(state => state.graph.toolbarState.showRank);
   const updateTime = useKialiSelector(state => state.graph.updateTime);
+  const externalServices = useKialiSelector(state => state.statusState.externalServices);
```

Comment:
I don't think `netobserv` would ever be found in `externalServices`. Is this leftover from an earlier iteration?

Reply:
Yea, I'm not sure why this came back into the code